offlineimap/imapserver.py

```diff
@@ -198,7 +198,7 @@ def __md5handler(self, response):
         challenge = response.strip()
         self.ui.debug('imap', '__md5handler: got challenge %s'% challenge)
 
-        passwd = self.__getpassword()
+        passwd = self.__getpassword().encode('ascii')
```

Comment:
I would rather put that change below into a call to `hmac.new`, which better shows the actual purpose.